packages/core/src/framebuffer/FramebufferSystem.js

```diff
@@ -216,14 +216,16 @@ export class FramebufferSystem extends System
      * @param {Number} g - Green value from 0 to 1
      * @param {Number} b - Blue value from 0 to 1
      * @param {Number} a - Alpha value from 0 to 1
+     * @param {PIXI.BUFFER_BITS} [mask=PIXI.BUFFER_BIT.COLOR] - Bitwise OR of masks
+     *  that indicate the buffers to be cleared, by default COLOR
      */
-    clear(r, g, b, a)
+    clear(r, g, b, a, mask = BUFFER_BITS.COLOR)
```

Comment:
default should be equivalent to: `gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT`

```suggestion
    clear(r, g, b, a, mask = BUFFER_BITS.COLOR | BUFFER_BITS.DEPTH)
```